plugins/modules/digital_ocean_certificate_info.py

```diff
@@ -87,8 +88,7 @@ def core(module):
         if status_code != 200:
             module.fail_json(msg="Failed to retrieve certificates for DigitalOcean")
 
-        resp_json = response.json
-        certificate = resp_json['certificate']
+        certificate = [response.json['certificate']]
```

Comment:
Perhaps we should test keys before assuming they're there to mitigate potential exceptions.

Reply:
That makes sense. Do you want to do this as part of this PR or a follow up though? Because I think that is a fairly common thing that probably needs addressed across all of the other modules as well.

Reply:
> That makes sense. Do you want to do this as part of this PR or a follow up though? Because I think that is a fairly common thing that probably needs addressed across all of the other modules as well.

Yep, that's a good point, we'll be doing lots of changes in 2.x -- e.g., I'd like to get rid of the redundant `digital_ocean` prefix (in addition to being redundant, "DigitalOcean" is actually one word). So, we can leave it alone for now.